### ISSUE TITLE
Compatibility with React 17?

### ISSUE THREAD

**Issue by kmacancela** [aautio/react-modal-image#58]
While using React v17.0.2, I am getting an issue with installing react-model-image. Error says it is unable to resolve dependency tree. Furthermore: 
```
While resolving: new@0.1.0
Found: react@17.0.2
node_modules/react
react@"^17.0.2" from the root project
Could not resolve dependency:
peer react@"16.x" from react-modal-image@2.5.0
node_modules/react-modal-image
react-modal-image@"*" from the root project
```

**Comment by xmany** [comment]
Someone please solve this

**Comment by aautio** [comment]
I've just published `react-modal-image@2.6.0` which fixes this issue. Sorry for the long wait!
